### PR TITLE
Fix up a number of problems with ansible/examples/builder.yml

### PR DESCRIPTION
**Pull request by dmick**
Note: this must be accompanied by changes to ceph-sepia-secrets (to add the jenkins api token, builder label lists, and better jenkins builder group lists: https://github.com/ceph/ceph-sepia-secrets/pull/817

### CODE REVIEW

File: ansible/examples/builder.yml [hunk 1]
```diff
@@ -882,7 +888,7 @@
 
     - name: Register Permanent Builder
       block:
-        - name: Register permenant builder using JNLP
+        - name: Register permanant builder using JNLP
```
Comment (zmc) [inline ansible/examples/builder.yml:891]:
```suggestion
        - name: Register permanent builder using JNLP
```

File: ansible/library/jenkins_node [hunk 2]
```diff
@@ -1,6 +1,3 @@
-#!/usr/bin/env python2
-#!/usr/bin/python2
-#!/usr/bin/env python3
```
Comment (zmc) [inline ansible/library/jenkins_node:3]: 🥴

File: ansible/examples/builder.yml (hunk 1 above)
Comment (zmc) [reply ansible/examples/builder.yml:891]: just absolutely egregious

File: ansible/ansible.cfg [hunk 3]
```diff
@@ -1,6 +1,7 @@
 [defaults]
 callback_plugins = callbacks
 retry_files_enabled = False
+vault_password_file = ~/.vault_pass.txt
```
Comment (zmc) [inline ansible/ansible.cfg:4]: Very happy about this one

File: ansible/examples/builder.yml [hunk 4]
```diff
@@ -919,6 +925,18 @@
...
+          service:
+            name: jenkins
+            state: stopped
+
+        - name: Kill any errant slave.jar or agent.jar processes
+          shell:
+            cmd: "pkill -f -9 'java.*(slave|agent).jar'"
+          register: result
+          become: true
+          failed_when: result.rc > 1
```
Comment (zmc) [inline ansible/examples/builder.yml:938]: Why `> 1` and not i.e. `> 0` ?

**Review by zmc — CHANGES_REQUESTED** [review]
Awesome! One change requested and one question

File: ansible/examples/builder.yml (hunk 1 above)
Comment (dmick) [reply ansible/examples/builder.yml:891]: for god's sake. thank you.

File: ansible/library/jenkins_node (hunk 2 above)
Comment (dmick) [reply ansible/library/jenkins_node:3]: I don't even

File: ansible/examples/builder.yml (hunk 4 above)
Comment (dmick) [reply ansible/examples/builder.yml:938]: 1 is "no processes matched", which isn't an error; I'm happy if there are no procs to kill.  2/3 are actual errors.

**Comment by dmick** [comment]
changed as requested.  I'm working on allowing the jenkins module to update an existing builder (for tags maintenance on existing builders) but that can be a followon PR

**Review by zmc — APPROVED** [review]

**Pull request merged**